src/Schedule.h

```diff
@@ -68,6 +68,23 @@ enum class TailStrategy {
     Auto
 };
 
+/** Different ways to handle the case when the start/end of the loops of stages
```

Comment:
Definitely need more detail, but I think putting examples in the comment on the compute_with directive is the best way to do that. (Can also link to a tutorial I suppose.)